alf/bin/play.py

```diff
@@ -133,9 +135,58 @@ def main(_):
         alf.close_env()
 
 
+def launch_snapshot_play(_):
+    """This play function uses historical ALF snapshot for playing a trained
+    model, consistent with the code snapshot that trains the model.
+
+    In the newer version of ``train.py``, a ALF snapshot is saved to ``root_dir``
+    right before the training begins. So this function prepends ``root_dir`` to
+    ``PYTHONPATH`` to allow using the snapshot ALF repo in that place.
+
+    Note that for any old training ``root_dir``, this won't change the ALF repo
```

Comment:
"old" => such,
old can mean an older version of the alf repo that's older than this commit.

Reply:
I assume that "newer" means the version updated by this PR, then it could be confusing after the PR is merged.

Reply:
I assume that "newer" means the version updated by this PR, then it could be confusing after the PR is merged.

Reply:
updated docstring